lib/Runtime/ByteCode/Scope.cpp

```diff
@@ -124,8 +124,10 @@ void Scope::MergeParamAndBodyScopes(ParseNode *pnodeScope, ByteCodeGenerator *by
             return;
         }
 
-        Assert(paramScope->m_symList == nullptr || paramScope->FindLocalSymbol(sym->GetName()) == nullptr);
-        paramScope->AddNewSymbol(sym);
+        if (paramScope->FindLocalSymbol(sym->GetName()) == nullptr)
```

Comment:
Can you explain the need to add arguments to both param and body scopes? There can only be one binding. Can we have it only in the param scope?